extensions/wikia/ExactTargetUpdates/tasks/ExactTargetWikiTaskHelper.php

```diff
@@ -66,7 +92,19 @@ public function prepareWikiDataExtensionForUpdate( $iCityId ) {
 		$aKeys = [
 			'city_id' => $iCityId,
 		];
-		
+
+		$aWikiData = $this->getWikiDataArray( $iCityId );
+		$aWikiDataExtension = [ 'DataExtension' => [] ];
+		$aWikiDataExtension['DataExtension'][] =  [
+				'CustomerKey' => $aCustomerKeys['city_list'],
+				'Keys' => $aKeys,
+				'Properties' => $aWikiData,
+		];
+
+		return $aWikiDataExtension;
+	}
+
+	public function getWikiDataArray( $iCityId ) {
 		/* Get wikidata from master */
 		$oWiki = \WikiFactory::getWikiById( $iCityId, true );
 		$aWikiData = [
```

Comment:
I'm not fetching all records here so probably it's better to have explicit list.